src/OrchardCore.Modules/OrchardCore.Media/Startup.cs

```diff
@@ -187,6 +188,8 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IMediaNameNormalizerService, NullMediaNameNormalizerService>();
 
             services.AddScoped<IUserAssetFolderNameProvider, DefaultUserAssetFolderNameProvider>();
+            services.AddTransient<IChunkFileUploadService, ChunkFileUploadService>();
```

Comment:
Maybe `AddSingleton()` so that all is resolved once, including the prefix if computed in the ctor, but as you want ;)